src/org/scada_lts/dao/model/multichangehistory/MultiChangeHistoryValues.java

```diff
@@ -0,0 +1,140 @@
+package org.scada_lts.dao.model.multichangehistory;
+
+import java.util.Objects;
+
+/**
+ * @author  grzegorz.bylica@abilit.eu on 16.10.2019
+ */
+public class MultiChangeHistoryValues {
+
+    private int id;
+    private int userId;
+    private String userName;
+    private String viewAndCmpIdentyfication;
+    private String interpretedState;
+    private long ts;
+    private int valueId;
+    private String value;
+    private int dataPointId;
+    private String xidPoint;
+
+    public MultiChangeHistoryValues() {}
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
+        this.userId = userId;
+    }
+
+    public String getUserName() {
+        return userName;
+    }
+
+    public void setUserName(String userName) {
+        this.userName = userName;
+    }
+
+    public String getViewAndCmpIdentyfication() {
+        return viewAndCmpIdentyfication;
+    }
+
+    public void setViewAndCmpIdentyfication(String viewAndCmpIdentyfication) {
+        this.viewAndCmpIdentyfication = viewAndCmpIdentyfication;
+    }
+
+    public String getInterpretedState() {
+        return interpretedState;
+    }
+
+    public void setInterpretedState(String interpretedState) {
+        this.interpretedState = interpretedState;
+    }
+
+    public long getTs() {
+        return ts;
+    }
+
+    public void setTs(long ts) {
+        this.ts = ts;
+    }
+
+    public int getValueId() {
+        return valueId;
+    }
+
+    public void setValueId(int valueId) {
+        this.valueId = valueId;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public int getDataPointId() {
+        return dataPointId;
+    }
+
+    public void setDataPointId(int dataPointId) {
+        this.dataPointId = dataPointId;
+    }
+
+    public String getXidPoint() {
+        return xidPoint;
+    }
+
+    public void setXidPoint(String xidPoint) {
+        this.xidPoint = xidPoint;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        MultiChangeHistoryValues that = (MultiChangeHistoryValues) o;
+        return id == that.id &&
```

Comment:
Is it possible to change name that to more understandable?

Reply:
This code is created automatically